Calculater/Program.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+
+namespace CalculateTheMonths
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
```

Comment:
Please use the new modern C# Template without namespace, class and Main method for modern .NET applications.